.mbedignore

```diff
@@ -0,0 +1 @@
+bare_metal_tests/*
```

Comment:
Instead of placing this in the root with `bare_metal_tests/*` could you place it in `bare_metal_tests` with `*` please? I don't want to set a precedent of having root level `.mbedignore` files just yet.

Reply:
Fixed directory tree as suggested. Verified that bare-metal test and regular tests can be successfully executed.